src/main/java/duke/Commands/Command.java

```diff
@@ -0,0 +1,174 @@
+package duke.Commands;
+
+import duke.Exception.DukeException;
+import duke.Tasks.Deadlines;
+import duke.Tasks.Event;
+import duke.Tasks.Task;
+import duke.Tasks.TaskList;
+import duke.Tasks.ToDos;
```

Comment:
I personally feel like it is uncommon to name a class using plural forms, because it is a bit strange if we say "create a ToDos instance"?